core/src/main/resources/reference.conf

```diff
@@ -370,20 +361,20 @@ database {
       }
     }
 
-    mysql_example {
-      driver = "slick.driver.MySQLDriver$"
-      db {
-        driver = "com.mysql.jdbc.Driver"
-        url = "jdbc:mysql://host/cromwell"
-        user = "user"
-        password = "pass"
-        connectionTimeout = 5000
-      }
-
-      migration {
-        read-batch-size = 100000
-        write-batch-size = 100000
-      }
-    }
+    //mysql_example {
```

Comment:
While you're in here, can you make the scheme for commenting out blocks uniform? For instance I see `#` above and `//` here